packages/react-wildcat/src/server.js

```diff
@@ -107,6 +107,18 @@ function start() {
                 }));
             });
 
+            // Allow custom middleware priority over react/jspm/render from below
+            // otherwise server-only routes will receive 404's.
+            if (appServerSettings.middleware && appServerSettings.middleware.length) {
+                appServerSettings.middleware.forEach(function eachCustomAppMiddleware(middlewareConfigFunction) {
+                    if (typeof middlewareConfigFunction === "function" && middlewareConfigFunction.length >= 1) {
```

Comment:
You're testing a function's length here, which is either incorrect or not actually testing anything of use.